src/app/pages/checkout-payment/checkout-payment/checkout-payment.component.ts

```diff
@@ -86,6 +94,11 @@ export class CheckoutPaymentComponent implements OnInit, OnChanges, OnDestroy {
       const redirect = params.get('redirect');
       this.redirectStatus = redirect;
     });
+
+    this.checkoutFacade.basketValidationResults$.pipe(takeUntil(this.destroy$)).subscribe(validationResults => {
```

Comment:
Try not to use subscribe, especially in very preventable cases like this. Instead create an observable stream that emits what you want (true when the basket has an error) and then subscribe to it directly in the template using the `async` pipe.